instrumentation-api-annotation-support/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ group = "io.opentelemetry.instrumentation"
 
 dependencies {
   implementation(project(":instrumentation-api"))
+  compileOnly(project(":instrumentation-api-caching"))
```

Comment:
Can you search for this string and copy in the same comment about the workaround?